arch/arm/Kconfig

```diff
@@ -1743,6 +1778,17 @@ config ARCH_SELECT_MEMORY_MODEL
 config HAVE_ARCH_PFN_VALID
 	def_bool ARCH_HAS_HOLES_MEMORYMODEL || !SPARSEMEM
 
+config ARCH_ENABLE_MEMORY_HOTPLUG
```

Comment:
Can potentially break QC support. If it is a Broadcom-specific config option (along with the other two), then set def_bool y only if BCM chips.